### PR TITLE
Issue/448

### PR DESCRIPTION
**Pull request by hayleyavw**
Added about pages:
- Releases
- Contributors
- Interactives

Tests have been written for the new URLs and Views, and all tests have been refactored for consistency and readability.

### CODE REVIEW

File: csfieldguide/general/urls.py [hunk 1]
```diff
@@ -5,6 +5,28 @@
 from . import views
 
 urlpatterns = [
-    url(r"^$", views.GeneralIndexView.as_view(), name="home"),
-    url(r"^about$", views.GeneralAboutView.as_view(), name="about"),
+    # e.g. csfieldguide.com/
```
Comment (JackMorganNZ) [inline csfieldguide/general/urls.py]: Should be `csfieldguide.org.nz`

File: csfieldguide/templates/base.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-  
+
```
Comment (JackMorganNZ) [inline csfieldguide/templates/base.html]: Reason for this?

File: csfieldguide/templates/chapters/index.html [hunk 3]
```diff
@@ -14,7 +14,7 @@ <h1 id="chapters" class="section-heading anchor-link">Chapters</h1>
 {% block content %}
 
   {% if all_chapters %}
-      
+
```
Comment (JackMorganNZ) [inline csfieldguide/templates/chapters/index.html]: Reason for extra whitespace? This seems to be making page size bigger for no gain.

File: csfieldguide/templates/general/releases.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  Releases
+{% endblock %}
+
+{% block page_heading %}
+  <h1 id="releases" class="section-heading anchor-link">Releases</h1>
+{% endblock page_heading %}
+
+{% block content %}
+
+<p>
```
Comment (JackMorganNZ) [inline csfieldguide/templates/general/releases.html]: Not indented within block.

File: csfieldguide/templates/interactives/index.html [hunk 5]
```diff
@@ -14,15 +14,16 @@ <h1 id="interactives" class="section-heading anchor-link">Interactives</h1>
 {% block content %}
 
   {% if all_interactives %}
-    
+
+    <!-- bug! does not show interactive objects in for loop -->
```
Comment (JackMorganNZ) [inline csfieldguide/templates/interactives/index.html]: This should be fixed.

**Comment by hayleyavw** [comment]
@JackMorganNZ This is now ready for rereview. The bug has not been addressed because it is out of the scope of this PR and has been a known bug for a while, instead I think an issue needs to made for it and can be bundled into the next interactives PR.

File: csfieldguide/templates/interactives/index.html (hunk 5 above)
Comment (JackMorganNZ) [reply csfieldguide/templates/interactives/index.html]: Now tracked in #538

**Pull request merged**